servicelayer/util.py

```diff
@@ -15,3 +16,19 @@ def backoff(failures=0):
 def service_retries():
     """A default number of tries to re-try an external service."""
     return range(30)
```

Comment:
Should we move the other helper functions from `memorious.model.common` here so they are all in one place?